frontend/src/components/Uploader/ServiceAreasUploader.tsx

```diff
@@ -60,27 +61,26 @@ function getCounties(serviceAreas: [string, string][]): string[] {
  * Aliases are compared to column names as lowerCase()
  */
 const COLUMNS = [
-  { aliases: ['CountyName', 'County Name', 'County'] },
-  { aliases: ['ZipCode', 'Zip Code', 'Zip'] }
-]
+  { aliases: ['CountyName', 'County Name', 'County'] }]
 
-let parse = parseRows<[string, string][], { state: State }>(COLUMNS, (([county, zip], rowIndex, { state }) => {
+let parse = parseRows<[string, string][], { state: State }>(COLUMNS, (([county], rowIndex, { state }) => {
 
   // Validate that zip exists
-  if (zip && !(zip in COUNTIES_BY_ZIP)) {
-    return new ParseError(rowIndex, 1, COLUMNS[1], `We don't support the zip code "${zip}" yet. Reach out to us at health@bayesimpact.org`)
-  }
+  // if (zip && !(zip in COUNTIES_BY_ZIP)) {
+  //   return new ParseError(rowIndex, 1, COLUMNS[1], `We don't support the zip code "${zip}" yet. Reach out to us at health@bayesimpact.org`)
+  // }
 
   // We infer missing zips and counties, so one zip might map to
   // more than one county, and one county maps to many zips.
-  let pairs = getCountyAndZip(state, county, zip)
+  // Note - we currenlt use '00000' as a placeholder in all counties.
+  let pairs = getCountyAndZip(state, county, '00000')
     .map(([county, zip]) => [county, zip] as [string, string])
 
   // Validate that counties exist
   let badCounty = pairs.value().find(([county]) => Object.keys(ZIPS_BY_COUNTY_BY_STATE[state]).indexOf(county!) <= -1)
 
   if (badCounty) {
-    return new ParseError(rowIndex, 0, COLUMNS[0], `We don't support county "${badCounty[0]}" yet. Reach out to us at health@bayesimpact.org.`)
+    return new ParseError(rowIndex, 0, COLUMNS[0], `We don't support "${badCounty[0]}" in ${capitalizeWords(state)} yet. Reach out to us at health@bayesimpact.org.`)
```

Comment:
Is the email address `health@bayesimpact.org` the same one used throughout the app? 

If not, please change for consistency.

Reply:
how many characters actually pop up in the snack bar? seems like you can just rid of the whole email address part